docs/getting-started/signing-transactions.md

```diff
@@ -3,86 +3,227 @@ sidebar_position: 3
 ---
 
 # Signing transactions
+Before being able to submit transactions, you (or the user who owns the account from which the funds are being spent) need to sign them. For each different way in which you can create a wallet and maintain an account, there is a way to sign transactions.
+
+For simplicity, in all of the following cases let us consider a transaction obtained from the following function:
+```js
+async function getTransactions(from) {
+    const response = await fetch('https://delegator-dev01b.arringo.co/api-test/CreateDelegatedPayment', {
+        method: 'POST',
+        body: JSON.stringify({
+            assetId: 'usdd',
+            sender: {
+                address: from,
+                amount: 5,
+            },
+            receiver: 'CNIOPGMZ3OSZLPEO7POGMVUEQVDV2RT3G4MSKGHRBX2SOPOEIOGBIFIMVQ',
+        }),
+    });
+    const {noSign, toSign} = await response.json();
+    return {noSign, toSign};
+}
+```
 
-This is a sample page.
-
-## Configure i18n
-
-Modify `docusaurus.config.js` to add support for the `fr` locale:
+## Signing with Algorand Wallet (WalletConnect)
+If you (or your users) are using the Algorand Wallet, WalletConnect is the protocol that you will be using. In this guide we will suppose that some user is interacting with a web page, and is prompted to sign a transaction.
 
-```js title="docusaurus.config.js"
-module.exports = {
-  i18n: {
-    defaultLocale: 'en',
-    locales: ['en', 'fr'],
-  },
-};
+To use walletconnect, we need to import two scripts: one for managing the connection with the wallet, and one to show a QR code:
+```html
+<script type="text/javascript" src="/img/https://github.com/WalletConnect/walletconnect-monorepo/releases/download/1.7.0/qrcode-modal.min.js"></script>
+<script type="text/javascript" src="/img/https://github.com/WalletConnect/walletconnect-monorepo/releases/download/1.7.0/client.min.js"></script>
+```
+Since Algorand uses a custom WalletConnect protocol, one last script which enables communication in that protocol is needed. This library is not available as a minified file, so we will be using Browserify.
+Firstly, we'll install json-rpc-tools and browserify with npm:
+```sh
+npm i @json-rpc-tools/utils browserify
```

Comment:
I think we can replace this library and the corresponding steps with just

```
wallet.sendCustomRequest({
    id: Date.now() + Math.floor(Math.random() * 1000),
    jsonrpc: "2.0",
    method: "algo_signTxn",
    params: [group],
})
```

Or define a formatting function somewhere. As library build instructions are definitely out of this guide.